File: src/main/java/org/artembogomolova/demo/webapp/config/RepositoryConfig.java
package org.artembogomolova.demo.webapp.config;

import org.springframework.context.annotation.Configuration;
import org.springframework.data.jpa.repository.config.EnableJpaRepositories;

@Configuration
@EnableJpaRepositories(basePackages = "org.artembogomolova.demo.webapp.dao")
public class RepositoryConfig {

}

File: app/src/main/kotlin/org/artembogomolova/demo/webapp/main/config/RepositoryConfig.kt
package org.artembogomolova.demo.webapp.main.config

import org.springframework.context.annotation.Configuration
import org.springframework.data.jpa.repository.config.EnableJpaRepositories

@Configuration
@EnableJpaRepositories(basePackages = ["org.artembogomolova.demo.webapp.main.dao.repo"])
class RepositoryConfig 